profiles/migrations/0025_fix_primary_emails.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.2.10 on 2020-05-06 11:46
+
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def fix_primary_emails(apps, schema_editor):
+    """
+    Goes through all the profiles:
+    1) Tries to set first matching email with primary=True as a primary email
+    2) Tries to set first email as a primary email
+    3) If no emails found, profile gets deleted with a message
+    4) If profiles has more than one primary email, first will remain as the primary and primary=False will be set for
+       the rest
+    """
+    Profile = apps.get_model("profiles", "Profile")
+    for profile in Profile.objects.all():
+        primary_emails = profile.emails.filter(primary=True)
+        primary_email = primary_emails.first()
+        if not primary_email:
+            if profile.emails.count() > 0:
+                primary_email = profile.emails.first()
+            else:
+                print(f"Removing profile of {profile.first_name} {profile.last_name}\n")
+                profile.delete()
+                continue
+            primary_email.primary = True
+            primary_email.save()
+        elif primary_emails.count() > 1:
```

Comment:
I think you need to make this `elif` an ìf`-clause to make the `4)` on the docstring happen.

Reply:
I think that `elif` works just fine and saves us one if statement check in 2/4 cases 😄 Wrote a test for that migration as I checked it.